src/components/Person.jsx

```diff
@@ -0,0 +1,46 @@
+import PropTypes from 'prop-types';
+import './person.scss';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        { name ? `My name is ${name}` : 'Unknown'}
```

Comment:
it's not necessary, all people have names

Reply:
Agree, name is required field

Reply:
Fixed